iOS-TvShows/iOS-TvShows/Common/Networking/Authentication/Model/AuthInfo.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  AuthModel.swift
+//  iOS-TvShows
+//
+//  Created by Filip Ban on 08.11.2021..
+//
+
+import Foundation
+
+struct AuthInfo: Codable {
+
+    let uid: String?
+    let accessToken: String?
+    let client: String?
+    let expiry: String?
+    let tokenType: String?
+    let contentType: String?
+
+    init(from response: HTTPURLResponse) {
+        uid = response.headers.dictionary["uid"]
```

Comment:
How come you aren't using constants values here?

Reply:
same as above, did it before constants